lg_common/scripts/browser_url.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+import rospy
+from std_msgs.msg import String
+from lg_common.msg import BrowserURL
+from lg_common.msg import AdhocBrowsers
+from interactivespaces_msgs.msg import GenericMessage
+
+
+def main():
+    rospy.init_node('browser_current_url', anonymous=False)
+
+    viewports = [ v.split('/')[-1] for v in rospy.get_param_names() if v.startswith('/viewport')]
+
+    url_service = BrowserURLService(viewports)
+    rospy.Subscriber('/browser/extension/current_url', BrowserURL, url_service.handle_url_message)
+
+
+    for viewport in viewports:
+        topic_name = '/browser_service/{}'.format(viewport)
+        rospy.Subscriber(topic_name,
+                         AdhocBrowsers,
+                         lambda msg: url_service.handle_browsers(msg, viewport))
+
+        rospy.Service('/browser/{}/current_url'.format(viewport),
```

Comment:
so for browsers that are emitting `url` on the `current_url` topic, we want to store a `list` where every new incmoming browser url is placed on the top of the stack under index 0.

```json
[
{ 
    "url": "http://yandex.ru",
    "ros_instance_id": "1203-49rtuih",
    "created_timestamp": "12345678"
   },
{ 
    "url": "http://yandex.ru",
    "ros_instance_id": "1203-49rtuih",
    "created_timestamp": "12345678"
   },
{ 
    "url": "http://google.com",
    "ros_instance_id": "1230i9rtujoe3k2",
    "created_timestamp": "12345678"
 }
]
```

^ implement such data representation in the adhoc browser pool
keep it in sync or use self.browsers() to reprent this data


for preloading in portal there's going to be an intermediate state with 2 browsers - make it return the newer one.

by default - return all browsers, make it possible to request `last` browser using service arguments.